application/frontend/controllers/UserController.php

```diff
@@ -4,17 +4,49 @@
 use common\models\User;
 use frontend\components\BaseRestController;
 use Yii;
-use yii\web\NotFoundHttpException;
 
 class UserController extends BaseRestController
 {
+    public function actionIndex() // GET /user
+    {
+        return User::find()->all();
+    }
+
+    public function actionView($employeeId) // GET /user/abc123
+    {
+        $user = User::findOne(['employee_id' => $employeeId]);
+
+        if (isset($user)) {
+            return $user;
+        }
+
+        Yii::$app->response->statusCode = 204;
+    }
+
     public function actionCreate(): User
     {
-        $existingUser = User::findOne([
-            'employee_id' => Yii::$app->request->getBodyParam('employee_id')
-        ]);
+        $user = new User();
+
+        $user->scenario = User::SCENARIO_NEW_USER;
+
+        $user->attributes = Yii::$app->request->getBodyParams();
+
+        parent::save($user);
```

Comment:
I did it that way to be more explicit (readability) about where the call was coming from but I don't need to...chalk that up to my learning PHP as well, I've seen `::` being used in ways I didn't think meant strictly static but I can see how it might.  Shouldn't something have complained about that syntactically or at least at runtime or is this the "forgiveness" of PHP?